src/main/java/com/textmessenger/controller/CommentController.java

```diff
@@ -29,27 +29,27 @@
   }
 
   @PostMapping("/post/{id}/user/{userId}")
-  ResponseEntity<?> createComment(@Valid @PathVariable("id") PostRxDto post,
-                                  @Valid @PathVariable("userId") UserRxDto user,
-                                  @Valid @RequestBody CommentRxDto comment) {
+  ResponseEntity<?> createComment(@Valid @PathVariable("id") Post post,
+                                  @Valid @PathVariable("userId") User user,
+                                  @Valid @RequestBody Comment comment) {
     commentService.createComment(post, user, comment);
     return ResponseEntity.ok().build();
   }
 
   @GetMapping("/post/{id}")
-  ResponseEntity<?> getAllCommentsFromPost(@Valid @PathVariable("id") PostRxDto post) {
+  ResponseEntity<?> getAllCommentsFromPost(@Valid @PathVariable("id") Post post) {
     return Optional.of(ResponseEntity.ok().body(commentService.findAllPostFromPost(post)))
             .orElse(ResponseEntity.noContent().build());
   }
 
   @PutMapping
-  ResponseEntity<?> updateComment(@Valid @RequestBody CommentRxDto comment) {
+  ResponseEntity<?> updateComment(@Valid @RequestBody Comment comment) {
     commentService.updateComment(comment);
     return ResponseEntity.ok().build();
   }
 
   @DeleteMapping
-  ResponseEntity<?> deleteComment(@Valid @RequestBody CommentRxDto comment) {
+  ResponseEntity<?> deleteComment(@Valid @RequestBody Comment comment) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_aSAJjEPbGfqJYx&open=AWWU5_aSAJjEPbGfqJYx)

[ISSUE_KEY]: AWWU5_aSAJjEPbGfqJYx